net/zabbix4/Portfile

```diff
@@ -20,13 +20,15 @@ long_description    Zabbix is the ultimate open source availability and \
                     of the best commercial ones. 
 
 array set VERSIONS {
-    4  4.0.20
+    4  4.0.24
     42 4.2.8
```

Comment:
The 42 line could be removed since you're removing the zabbix42 subports.

Reply:
I've left it so the obsolete check can redirect users.